pmd-xml/src/main/resources/category/xsl/performance.xml

```diff
@@ -16,7 +16,7 @@ Rules that flag suboptimal code.
           class="net.sourceforge.pmd.lang.rule.XPathRule"
           externalInfoUrl="${pmd.website.baseurl}/pmd_rules_xsl_performance.html#avoidaxisnavigation">
         <description>
-Avoid using the 'following' or 'preceeding' axes whenever possible, as these can cut
+Avoid using the 'following' or 'preceding' axes whenever possible, as these can cut
 through 100% of the document in the worst case.  Also, try to avoid using 'descendant'
 or 'descendant-self' axes, as if you're at the top of the Document, it necessarily means
```

Comment:
```suggestion
or 'descendant-or-self' axes, as if you're at the top of the Document, it necessarily means
```

Not directly related, but this is the real name of the axis

Reply:
@oowekyala done.